src/Libraries/PythonNodeModelsWpf/ScriptEditorWindow.xaml.cs

```diff
@@ -131,6 +132,17 @@ private void OnTextAreaTextEntered(object sender, TextCompositionEventArgs e)
 
         #region Private Event Handlers
 
+        private void OnNodeModelCodeMigrated(object sender, EventArgs e)
+        {
+            var args = e as PythonCodeMigrationEventArgs;
+            if (args is null)
+                throw new NullReferenceException(nameof(args));
```

Comment:
Avoid throwing exceptions on methods outside of node execution as these will likely crash Dynamo